### PR TITLE
chore: improve mobile layout on about us page

### PR DESCRIPTION
**Pull request by argshook**
This PR includes

1) refactor code for easier reuse
2) improve mobile responsiveness in "About Us" page
3) add "Central Group" logo on "About Us" page
4) change team picture to video of team zoom call 

---

- add Section component
- Section: use `row` instead of `column`
- remove usage of `seCss`
- move ProductMetrics css to component
- add central-group logo to used by section
- reorder investors list to keep original order when rendered
- improve investors and team block responsiveness on mobile

### CODE REVIEW

File: src/components/ProductMetrics/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.root {
+  --columns: 1fr;
```
Comment (argshook) [inline src/components/ProductMetrics/styles.module.css:2]:
```suggestion
  --columns: 1;
```

**Comment by pswu11** [comment]
Screenshot 2022-07-26 at 15 01 18
can update to 1,400+ now ... don't hate me :D

**Comment by argshook** [comment]
> Screenshot 2022-07-26 at 15 01 18
> 
> can update to 1,400+ now ... don't hate me :D

thanks for pointing it out, @pswu11, updated!

**Comment by jerrinot** [comment]
the "hiring" button + text looks a bit odd on mobile. I cannot tell what's wrong with that, but it _feels_ off:
image

**Comment by argshook** [comment]
all good, @jerrinot, nice catch, thanks!

I pushed a fix to address that

**Review by insmac — APPROVED** [review]

**Pull request merged**